common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/azureactivedirectoryb2c/AzureActiveDirectoryB2CAuthorizationRequest.java

```diff
@@ -48,15 +48,20 @@ public AzureActiveDirectoryB2CAuthorizationRequest(final String responseType,
                                                        @NonNull final String clientId,
                                                        final String redirectUri,
                                                        final String state,
-                                                       final Set<String> scope,
+                                                       final String scope,
                                                        @NonNull final URL authority,
-                                                       @NonNull final String authorizationEndpoint,
                                                        final String loginHint,
                                                        final UUID correlationId,
                                                        final PkceChallenge pkceChallenge,
                                                        final String extraQueryParam,
                                                        final String libraryVersion) {
-        super(responseType, clientId, redirectUri, state, scope, authority, authorizationEndpoint,
+        super(responseType, clientId, redirectUri, state, scope, authority,
                 loginHint, correlationId, pkceChallenge, extraQueryParam, libraryVersion);
     }
+
+    @Override
+    public String getAuthorizationEndpoint() {
+        return null;
```

Comment:
Is this a `//TODO` ?

Reply:
Yes.  We haven't fully implemented anything for B2C yet.